compiler/rustc_span/src/symbol.rs

```diff
@@ -1147,6 +1148,7 @@ symbols! {
         skip,
         slice,
         slice_alloc,
+        slice_len_fn, // for a lowering of a `core::slice::len` in MIR
```

Comment:
```suggestion
        slice_len_fn,
```
the other symbols don't have an explanation where they are used either.